### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by omarIbrahim2** [livewire/livewire#4944] [1. Help]
im trying to update user profile when i update any input error appear in console 
**profile:1 Uncaught (in promise) 
Livewire Error:

Cannot find parent element in DOM tree containing attribute: [wire:id].

Usually this is caused by Livewire's DOM-differ not being able to properly track changes.

Reference the following guide for common causes: https://laravel-livewire.com/docs/troubleshooting 

Referenced element:

<i class="fa-solid fa-user"></i>**

`<div class="from-container">
                                <form wire:submit.prevent="update">
                                <div class="contact-form black-contact-form">
                                        <div class="input-item me-auto ms-0">
                                            <div>
                                                @error("name")

                                                <div class="alert alert-danger">
                                                    {{$message}}
                                                </div>

                                                @enderror
                                            </div>
                                            <input type="text" wire:model="name" placeholder="name">
                                            <span>
                                                <i class="fa-solid fa-user"></i>
                                            </span>
                                             <span class="second">
                                                <i class="fa-solid fa-pen"></i>
                                            </span>
                                        </div>

                                        <div class="input-item me-auto ms-0">
                                            <div>
                                                @error("email")

                                                <div class="alert alert-danger">
                                                    {{$message}}
                                                </div>

                                                @enderror
                                            </div>
                                            <input type="email" wire:model="email" placeholder="email">
                                            <span>
                                                <i class="fa-regular fa-envelope"></i>
                                            </span>
                                            <span class="second">
                                                <i class="fa-solid fa-pen"></i>
                                            </span>
                                        </div>

                                        <div class="input-item me-auto ms-0">
                                            <div>
                                                @error("password")

                                                <div class="alert alert-danger">
                                                    {{$message}}
                                                </div>

                                                @enderror
                                            </div>
                                            <input type="password" wire:model="password" placeholder="password"
                                                value="">
                                            <span>
                                                <i class="fa-solid fa-lock"></i>
                                            </span>
                                            <span class="second show-passowrd">
                                                <i class="fa-regular fa-eye-slash fa-flip-horizontal"></i>
                                            </span>
                                        </div>
                                        {{-- <div class="input-item me-auto ms-0">
                                            <input type="password"  placeholder="Confirm Password"
                                                value="">
                                            <span>
                                                <i class="fa-solid fa-lock"></i>
                                            </span>
                                            <span class="second show-passowrd">
                                                <i class="fa-regular fa-eye-slash fa-flip-horizontal"></i>
                                            </span>
                                        </div> --}}

                                        <div class="input-item me-auto m-0">
                                            <div>
                                                @error("phone")

                                                <div class="alert alert-danger">
                                                    {{$message}}
                                                </div>

                                                @enderror
                                            </div>
                                            <input type="tel" wire:model="phone" placeholder="phone" >
                                            <span>
                                                <i class="fa-solid fa-phone"></i>
                                            </span>
                                            <span class="second">
                                                <i class="fa-solid fa-pen"></i>
                                            </span>
                                        </div>
                                        {{-- <div class="input-item me-auto ms-0">
                                            <input type="file"  id="parent-profile">

                                            <label class="file-input" for="parent-profile">
                                                <span>
                                                    <i class="fa-regular fa-image"></i>
                                                </span>
                                                <p class="upload-text">Upload Profile</p>
                                                <button class="btn-custom btn-gray-custom">
                                                    choose
                                                </button>
                                            </label>
                                        </div> --}}

                                        <div class="input-item me-auto ms-0">
                                            <button  type="submit" class="custom-out-btn btn-form">
                                                save
                                            </button>
                                        </div>

                                </div>
                            </form>
                            </div>`
  

`

    public $name;
    public $email;
    public $phone;
    public $password;
    public $confirmPassword;

    protected $rules= [
      'name' => 'required|string',
      'email' => 'required|email',
      'phone' => 'required|string|max:8',
      'password' =>'required|string|max:9',
    ];

    public function mount(){

        $this->name = auth("parent")->user()->name;
        $this->email = auth("parent")->user()->email;
        $this->phone = auth("parent")->user()->phone;
    }

    public function render()
    {
        return view('livewire.parent-profile');
    }

    public function update()
    {
        $this->validate();

       $authParent = auth('parent')->user();

       $authParent->update([
        'name' => $this->name,
        'email' => $this->email,
        'phone' => $this->phone,
        'password'=> Hash::make($this->password),
       ]);

        return session()->flash('message' , "profile updated successfully");
    }

``

**Comment by bkilshaw** [comment]
Sounds like it might be malformed HTML; can you share the whole blade template? Looks like you only copy pasted a chunk of it.
